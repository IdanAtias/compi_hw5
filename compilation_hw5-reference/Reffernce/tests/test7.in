AGORA nedOrton = 8 NIS * 6 * 2 / 16 + 2 NIS;
	while(nedOrton > 1 NIS){
		print "#16) I know it's hard to fall.";
		break;
		print "Error #14";
	}
	while(nedOrton > 1 NIS)
	{
		{
			{
				while(nedOrton == 8 NIS)
				{
					print "#17) Confided in me was your heart.";
					nedOrton = 14 NIS / 2;
				}
				nedOrton = nedOrton - 1 NIS;
				bool ruliaJoberts = nedOrton*5 >= 30 NIS and not false;
				while   ( ruliaJoberts)
					{ print "#18) I know it's hurting you, but it's killing me.'"; break;}

				if(nedOrton >= 400 AGORA)
				{
					if(nedOrton <= 4 NIS)
						print "#21) now you're perfecting a world... meant to sin.";
					else
					{
						if(nedOrton <= 5 NIS)
							print "#20) our time is spent constructing";
						else
							print "#19) Nothing will last in this life";
					}
				}
				else
				{
					if((nedOrton != nedOrton / 5)
							and (100 AGORA * 2 + 1 NIS == nedOrton))
						print "#22) Constrict your hands around me,";
						
					NIS lenniferJopez = 1 NIS;
					if		(nedOrton != 3 NIS)
						if		((1 == nedOrton / lenniferJopez)  )
							print "#23) squeeze till I cannot breathe";
						else
							print "#24) this air tastes dead inside me,";
				}
			}
		}
	}