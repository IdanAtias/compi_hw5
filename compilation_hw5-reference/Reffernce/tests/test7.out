#16) I know it's hard to fall.
#17) Confided in me was your heart.
#18) I know it's hurting you, but it's killing me.'
#19) Nothing will last in this life
#20) our time is spent constructing
#21) now you're perfecting a world... meant to sin.
#22) Constrict your hands around me,
#24) this air tastes dead inside me,
#23) squeeze till I cannot breathe
